app/views/organizations/select_organization.html.erb

```diff
@@ -0,0 +1,63 @@
+<div class='table-responsive'>
+  <table class='table table-hover table-condensed users panel'>
```

Comment:
are the classes `users panel` doing something special? If not, I'd propose to remove them

Reply:
the users class does nothing but the panel class format the table.

Reply:
ok, let's remove the `users` class then